Tests/UIPilotTests/UIPilotTests.swift

```diff
@@ -1,10 +1,59 @@
+import SwiftUI
 import XCTest
 @testable import UIPilot
 
 final class UIPilotTests: XCTestCase {
-    func testExample() throws {
-        // This is an example of a functional test case.
-        // Use XCTAssert and related functions to verify your tests produce the correct
-        // results.
+    
+    enum PageName {
+        case first, second, third
+    }
+    
+    func testPaths() throws {
+        let pilot = UIPilot<PageName>(initial: .first)
+
+        let _ = UIPilotHost(pilot) { route in
```

Comment:
Maybe we can remove this as we are not using Host anywhere?

Reply:
Although I didn't use the UIPilotHost view directly, I did use it to populate the `routeMap` for these tests:
```swift
        XCTAssertNotNil(pilot.routeMap?(.first))
        XCTAssertNotNil(pilot.routeMap?(.second))
        XCTAssertNotNil(pilot.routeMap?(.third))
```